web/ui/navbar/index.svelte

```diff
@@ -104,7 +104,7 @@
     id="navbar_submenu"
     class="navbar_submenu"
     aria-hidden="true"
-    role="menu"
```

Comment:
Why we have menubar here if the submenu list is vertical?

Reply:
Probably my bad, going to revert it, thank for gotcha this